src/Storages/StorageGenerateRandom.cpp

```diff
@@ -707,7 +707,6 @@ Pipe StorageGenerateRandom::read(
         size_t estimated_block_size_bytes = 0;
         if (common::mulOverflow(max_block_size, estimated_row_size_bytes, estimated_block_size_bytes))
             throw Exception(ErrorCodes::TOO_LARGE_ARRAY_SIZE, "Too large estimated block size in GenerateRandom table: its estimation leads to 64bit overflow");
-        chassert(estimated_block_size_bytes != 0);
```

Comment:
In case of the empty `Tuple`, we will get a zero. And it is not a problem, because we won't enter the branch on l.712 anyway.